device/bgp/bgpsrtev4policy.yaml

```diff
@@ -101,6 +101,24 @@ components:
         ext_communities:
           description: >-
             Optional Extended Community settings.
+            The Extended Communities Attribute is a transitive optional BGP attribute, with the Type Code 16. Community and Extended Communities  attributes
+            are utilized to trigger routing decisions, such as acceptance, rejection,  preference, or redistribution. An extended community is an 8-Bytes value.
+            It is also divided into two main parts. The first 2 Bytes of the community encode a type and sub-type fields and the last 6 Bytes carry a unique set
```

Comment:
It is also divided ... -> It is divided ..

Reply:
updated